framework/persistence/infrastructure/seed.py

```diff
@@ -22,7 +22,7 @@ async def seed_initial_data_async(persistence: IPersistenceContext):
     persistence.add(merchant_one)
     persistence.add(merchant_two)
 
-    product_one = generate_entity(Product)
+    product_one = generate_entity(Product, True)
```

Comment:
What does the True argument do?

Reply:
the second argument determines whether or not to follow through the navigations and recursively generate entities

Reply:
when using generate entity, everything you set on the generated entity is just an override/replacement for what got generated